.github/workflows/ci.yml

```diff
@@ -14,12 +14,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['23', '24', '25']
-        rebar3: ['3.16.1']
+        otp: ['24', '25', '26', '27']
```

Comment:
Re-adding comment, we can still test 23, but IMHO we should drop it, and 24, though I left that.